<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamWidget</class>
 <widget class="QWidget" name="StreamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="iconImage"/>
     </item>
     <item>
      <widget class="QLabel" name="boldNameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ElidingLabel" name="nameLabel">
       <property name="text">
        <string>Device Title</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="directionLabel">
       <property name="text">
        <string>direction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deviceButton">
       <property name="text">
        <string>device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="muteToggleButton">
       <property name="toolTip">
        <string>Mute audio</string>
       </property>
       <property name="icon">
        <iconset theme="audio-volume-muted"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lockToggleButton">
       <property name="toolTip">
        <string>Lock channels together</string>
       </property>
       <property name="icon">
        <iconset theme="changes-prevent-symbolic"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="channelsGrid"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidingLabel</class>
   <extends>QLabel</extends>
   <header>elidinglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
